### PR TITLE
Multi Screen support with fix 

### PR DESCRIPTION
**Pull request by liberodark**
That is fork of work but have fix many errors
https://github.com/meskarune/i3lock-fancy/pull/147

### CODE REVIEW

**Comment by fubarhouse** [comment]
Any ideas when this is planned to be complete?
I might also be able to help with testing...

File: i3lock-fancy [hunk 1]
```diff
@@ -79,7 +79,7 @@ while true ; do
             esac ;;
         -t|--text) text=$2 ; shift 2 ;;
         -l|--listfonts)
-	    convert -list font | awk -F: '/Font: / { print $2 }' | sort -du | command -- ${PAGER:-less}
+	    convert -list font | awk -F: '/Font: / { print $2 }' | sort -du | command -- "${PAGER:-less}"
```
Comment (AladW) [inline i3lock-fancy:82]: Unrelated change.

File: i3lock-fancy [hunk 2]
```diff
@@ -118,8 +118,34 @@ else #black
...
+    if [[ "$LINE" =~ ([0-9]+)x([0-9]+)\+([0-9]+)\+([0-9]+) ]]; then
+        w=${BASH_REMATCH[1]}
+        h=${BASH_REMATCH[2]}
+        Xoff=${BASH_REMATCH[3]}
+        Yoff=${BASH_REMATCH[4]}
+        midx=$((w / 2 + Xoff - 60  / 2))
+        midy=$((h / 2 + Yoff - 60  / 2))
+        lock+=("$icon" -geometry "+$midx""+$midy" -composite)
+    fi
+done <<<"$(xrandr)"
```
Comment (AladW) [inline i3lock-fancy:132]: `< <(xrandr)`

File: i3lock-fancy [hunk 3]
```diff
@@ -118,8 +118,34 @@ else #black
...
+    if [[ "$LINE" =~ ([0-9]+)x([0-9]+)\+([0-9]+)\+([0-9]+) ]]; then
+        w=${BASH_REMATCH[1]}
+        h=${BASH_REMATCH[2]}
+        Xoff=${BASH_REMATCH[3]}
+        Yoff=${BASH_REMATCH[4]}
+        midx=$((w / 2 + Xoff - 60  / 2 - 95))
+        midy=$((h / 2 + Yoff - 60  / 2 + 160))
+        lock2+=(-annotate "+$midx""+$midy" "$text" )
+    fi
+done <<<"$(xrandr)"
```
Comment (AladW) [inline i3lock-fancy:145]: `< <(xrandr)`
